docs/reference/commandline/service_create.md

```diff
@@ -137,13 +137,145 @@ $ docker service create \
+- **`rprivate`**: The default. The same as `private`, meaning that no mount points
+                  anywhere within the original or replica mount points propagate
+                  in either direction.
+
+For more information about bind propagation, see the
+[Linux kernel documentation for shared subtree](https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt).
+
+#### Options for Volume Mounts
+The following options can only be used for named volumes (`type=volume`);
+
+| Option                | Description
+|:----------------------|:--------------------------------------------------------------------------------------------------------------------
+| **volume-driver**     | Name of the volume-driver plugin to use for the volume. Defaults to ``"local"``, to use the local volume driver to create the volume if the volume does not exist.
+| **volume-label**      | One or more custom metadata ("labels") to apply to the volume upon creation. For example, `volume-label=mylabel=hello-world,my-other-label=hello-mars`. For more information about labels, refer to [apply custom metadata](../../userguide/labels-custom-metadata.md).
+| **volume-nocopy**     | By default, if you attach an empty volume to a container, and files or directories already existed at the mount-path in the container (`dst`), the Engine copies those files and directories into the volume, allowing the host to access them. Set `volume-nocopy` to disables copying files from the container's filesystem to the volume and mount the empty volume.<br /><br />A value is optional:<ul><li>`true` or `1`: Default if you do not provide a value. Disables copying.</li><li>`false` or `0`: Enables copying.</li></ul>
+| **volume-opt**        | Options specific to a given volume driver, which will be passed to the driver when creating the volume. Options are provided as a comma-separated list of key/value pairs, for example, `volume-opt=some-option=some-value,some-other-option=some-other-value`. For available options for a given driver, refer to that driver's documentation.
+
+#### Differences between "--mount" and "--volume"
+
+The `--mount` flag supports most options that are supported by the `-v` /
+`--volume` flag for `docker run`. These are the differences:
+
+- The `--mount` flag allows you to specify a volume driver and volume driver
+  options *per volume*, without creating the volumes in advance. In contrast,
+  `docker run` only allows you to specify a single volume driver which is shared
+  by all volumes, using the `--volume-driver` flag).
+- The `--mount` flag allows you to specify custom metadata ("labels") for a volume,
+  before the volume is created.
+- When you use `--mount` with `type=bind`, the host-path must refer to an *existing*
+  path on the host. The path will not be created for you and the service will fail
+  with an error if the path does not exist.
+- The `--mount` flag does not allow you to relabel a volume with `Z` or `z` to specify
```

Comment:
Z/z are for selinux labeling. It is supported on `-v`, but doesn't make much sense on mounts since the dir must exist prior to mount any way, whereas `-v` will create it if it doesn't exist.